cmd/main.go

```diff
@@ -3,15 +3,28 @@ package cmd
 import (
 	"fmt"
 	"github.com/mitchellh/cli"
-	"os"
 	"os/exec"
 	"strings"
 )
 
 var execCommand = exec.Command
 
+type UiErrorWriter struct {
+	Ui cli.Ui
+}
+
+func (w *UiErrorWriter) Write(p []byte) (n int, err error) {
+	n = len(p)
+	if n > 0 && p[n-1] == '\n' {
+		p = p[:n-1]
+	}
+
+	w.Ui.Error(string(p))
+	return n, nil
+}
+
 func logCmd(cmd *exec.Cmd, ui cli.Ui, output bool) {
-	cmd.Stderr = os.Stderr
+	cmd.Stderr = &UiErrorWriter{ui}
```

Comment:
can be removed?

Reply:
Back to `os.Stderr`?  Without the ErrorWriter none of this works though unless I'm missing something.

`mockUi.ErrorWriter.String()` won't have anything in it

Reply:
I am Tang Doofus.

I didn't aware of `cli.Ui` doesn't provide getters for `Writer` & `ErrorWriter` and `MockUi` uses differnt names.

Also tried to implement trellis-cli's own UI struct. Doesn't seems worth it.

Please forget me.